### PR TITLE
Code gen the route partial isos

### PR DESCRIPTION
**Pull request by mbrandonw**
We've already got sourcery in the build process, might as well use it!

### CODE REVIEW

**Comment by mbrandonw** [comment]
> I like it! It'll make the template messier, but what do ya think of formatting it in such a way that the output is indented normally?

yahhh, was on the fence. it seems most people style their stencils so that outputs are optimal, but the stencil is the thing we actually have to edit, so I preferred styling it instead :/

we could prob run `swiftlint autocorrect` on the file to clean up some things, and also lib/Syntax will be coming soon and fix all of our problems!

**Comment by stephencelis** [comment]
> yahhh, was on the fence. it seems most people style their stencils so that outputs are optimal, but the stencil is the thing we actually have to edit, so I preferred styling it instead :/

I was probably overthinking, but I considered if folks decided to use this lib and codegen we'd optimize for readability. I don't think we need to worry about this right now.

File: .sourcery-templates/RoutePartialIsos.stencil [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import ApplicativeRouter
+import Prelude
+
+{% for enum in types.implementing.DerivePartialIsos|enum %}
+  extension {{ enum.name }} {
+    enum iso {
+      {% for case in enum.cases %}
+          {% if case.hasAssociatedValue %}
```
Comment (stephencelis) [inline .sourcery-templates/RoutePartialIsos.stencil]: This is a level off tho!

Comment (mbrandonw) [reply .sourcery-templates/RoutePartialIsos.stencil]: lol dangit

**Comment by mbrandonw** [comment]
> I was probably overthinking, but I considered if folks decided to use this lib and codegen we'd optimize for readability. I don't think we need to worry about this right now.

well right now we're in `pointfreeco` not `swift-web`! if we extract out to `swift-web` we can generalize and clean up a bit fer sure.

**Pull request merged**